core-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -136,3 +136,5 @@ clouddebug.stash.local.changes.and.sync=Stash local changes and sync to {0}
 clouddebug.unknown.repository.type=Unknown
 clouddebug.helpurl=
 clouddebug.snapshots=Cloud Debugger Snapshots
+clouddebug.stop.and.create.new.session=A cloud debugger session is already running.\nWould you like to stop it and start a new one?
```

Comment:
"new one" here, "new session" below. Not sure which I prefer, but please be consistent.